### PR TITLE
(dev/aws) add indexer resource

### PR DESCRIPTION
**Pull request by shangardezi**

### CODE REVIEW

**Review by Jarota — APPROVED** [review]

File: dev-aws/kafka-shared/iam_credentials.tf [hunk 1]
```diff
@@ -21,3 +21,33 @@ module "iam_credentials_producer" {
 
   cert_common_name = "auth-customer/credentials-api"
 }
+
+
```
Comment (sbuliarca) [inline dev-aws/kafka-shared/iam_credentials.tf:25]: Can you please use the consumer module?
Example: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared/customer-billing.tf#L23-L30

Comment (shangardezi) [reply dev-aws/kafka-shared/iam_credentials.tf:33]: https://github.com/utilitywarehouse/kafka-cluster-config/pull/72/commits/a8c6fcc5d4cbbd030be3daaf63947307abbff507

File: dev-aws/kafka-shared/iam_credentials.tf [hunk 2]
```diff
@@ -21,3 +21,41 @@ module "iam_credentials_producer" {
...
+module "iam_credentials_consumer" {
+  source = "../../modules/consumer"
+
+  topic          = kafka_topic.iam_credential_v1.name
+  consumer_group = "indexer-iam-credentials-v1"
+
+  cert_common_name = "auth-customer/iam-credentials-v1-indexer"
+}
+
+resource "kafka_acl" "indexer_iam_credentials_v1_topic_access" {
```
Comment (sbuliarca) [inline dev-aws/kafka-shared/iam_credentials.tf:34]: By using the consumer module, we don't really need these individual resources, as they're included in the module

Comment (shangardezi) [reply dev-aws/kafka-shared/iam_credentials.tf]: https://github.com/utilitywarehouse/kafka-cluster-config/pull/72/commits/4d6fb6630ae2f8ab6b2e0f9fa3653d69c8460b34

**Review by Nesze — APPROVED** [review]

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**